terraform/custom_domains/environment_domains/workspace_variables/afqts_dev.tfvars.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Can we make the file names in workspace variables directory match the naming convention in the AKS workspace variables directory?

We don't need the `afqts_` prefix as we're already in this project, and we've gone with full names like `development` and `preproduction`.

Reply:
File names updated and makefile code updated to reflect filename changes.